app/views/onestring.php

```diff
@@ -23,13 +23,17 @@
     <tr>
         <th>Locale</th>
         <th>Translation</th>
+        <th>&nbsp;</th>
     </tr>
     <?php
+    $reference_locale = Project::getReferenceLocale($repo);
```

Comment:
Maybe you don't need a temporary variable here since you only use it once later?